myapp/spec/requests/task_spec.rb

```diff
@@ -0,0 +1,93 @@
+require 'rails_helper'
+
+RSpec.describe 'Tasks', type: :request do
+  describe 'GET /index' do
+    let(:tasks) { create_list(:task, 10) }
+
+    it 'renders a successful response' do
+      get tasks_url
+      expect(response).to have_http_status(:ok)
+    end
+  end
+
+  describe 'GET /new' do
+    it 'renders a successful response' do
+      get new_task_url
+      expect(response).to have_http_status(:ok)
+    end
+  end
+
+  describe 'GET /edit' do
+    let(:task) { create(:task) }
+
+    it 'renders a successful response' do
+      get edit_task_url(task)
+      expect(response).to have_http_status(:ok)
+    end
+  end
+
+  describe 'POST /create' do
+    context 'with parameters' do
+      let(:params) do
+        { task: {
+          title: 'task',
+          priority: 'low',
+          status: 'working',
+          description: 'task desu',
+          expires_at: Time.now + 1.week
+        } }
+      end
+      it 'creates a new Task' do
+        expect do
+          post tasks_url, params:
+        end.to change(Task, :count).by(1)
```

Comment:
expectにはブロック渡せるのでこんな感じで1行で書けますよー

```suggestion
        expect { post tasks_url, params: params }.to change(Task, :count).by(1)
```

Reply:
これくらいなら1ライナーで書いたほうがよさそうですね！
こちら修正して他の箇所にも適用しました。